sql/Pg-custom_tables.sql

```diff
@@ -152,4 +152,7 @@ create table debits (id serial, reference text, description text, transdate date
 create table credits (id serial, reference text, description text, transdate date, accno text, amount numeric(12,2));
 create table debitscredits (id serial, reference text, description text, transdate date, debit_accno text, credit_accno text, amount numeric(12,2));
 
+ALTER TABLE bank ADD qriban TEXT;
```

Comment:
don't we have to place this into a version sql? Or how is it executed?

Reply:
Yes, you are right. This was for the development phase part. I create the version sql file to update databases automatically and then you can merge it.